hooks/useLatestWeatherForecasts.ts

```diff
@@ -40,28 +43,58 @@ interface WindSpeed {
   speed: string;
 }
 
-interface ZoneForecast {
-  label: string; // TODO: tighten this, it's either a day of the week, or a day with "Night" appended
+type PeriodLabel =
+  | 'Sunday'
+  | 'Monday'
+  | 'Tuesday'
+  | 'Wednesday'
+  | 'Thursday'
+  | 'Friday'
+  | 'Saturday'
+  | 'Sunday Night'
+  | 'Monday Night'
+  | 'Tuesday Night'
+  | 'Wednesday Night'
+  | 'Thursday Night'
+  | 'Friday Night'
+  | 'Saturday Night';
+
+export interface ZoneForecastPeriod {
+  // TODO: does this object need an actual time? Right now, all it has is a relative field like "Sunday Night",
```

Comment:
We should draw a line in the sand for the amount of effort put into this particular take on this approach - e.g. instead of attempting to parse "Sunday Night" into a UTC date-time, we can just assert that the JSON API give us a UTC date-time.